planning/behavior_path_planner/autoware_behavior_path_planner_common/test/test_turn_signal.cpp

```diff
@@ -318,20 +322,20 @@ TEST(BehaviorPathPlanningTurnSignal, Condition2)
       autoware::motion_utils::findFirstNearestSegmentIndexWithSoftConstraints(
         path.points, current_pose, 3.0, 1.0);
     const auto result_signal = turn_signal_decider.resolve_turn_signal(
-      path, current_pose, current_seg_idx, intersection_signal_info, behavior_signal_info,
-      nearest_dist_threshold, nearest_yaw_threshold);
+      path, current_pose, current_seg_idx, intersection_signal_info, roundabout_signal_info,
+      behavior_signal_info, nearest_dist_threshold, nearest_yaw_threshold);
     EXPECT_EQ(result_signal.command, TurnIndicatorsCommand::ENABLE_LEFT);
   }
 
-  // current pose is on the intersection desired end
+  // current pose is right on the intersection desired end
   {
     Pose current_pose = generateEgoSamplePose(65.0f, 0.0f, 0.0);
     const size_t current_seg_idx =
       autoware::motion_utils::findFirstNearestSegmentIndexWithSoftConstraints(
         path.points, current_pose, 3.0, 1.0);
     const auto result_signal = turn_signal_decider.resolve_turn_signal(
-      path, current_pose, current_seg_idx, intersection_signal_info, behavior_signal_info,
-      nearest_dist_threshold, nearest_yaw_threshold);
+      path, current_pose, current_seg_idx, intersection_signal_info, roundabout_signal_info,
+      behavior_signal_info, nearest_dist_threshold, nearest_yaw_threshold);
     EXPECT_EQ(result_signal.command, TurnIndicatorsCommand::ENABLE_LEFT);
```

Comment:
Sorry if you are already noticed. According to tests on my local machine, the returned `result_signal.command` showed `TurnIndicatorsCommand::ENABLE_RIGHT`. So the new feature would be causing reversed turn signal in this test case. Could you kindly check the possible cause?